agent/ecscni/plugin.go

```diff
@@ -290,13 +290,14 @@ func (client *cniClient) constructNetworkConfig(cfg interface{}, plugin string)
 
 func (client *cniClient) createENINetworkConfig(cfg *Config) (string, *libcni.NetworkConfig, error) {
 	eniConf := ENIConfig{
-		Type:                 ECSENIPluginName,
-		CNIVersion:           client.cniVersion,
-		ENIID:                cfg.ENIID,
-		IPV4Address:          cfg.ENIIPV4Address,
-		MACAddress:           cfg.ENIMACAddress,
-		IPV6Address:          cfg.ENIIPV6Address,
-		BlockInstanceMetdata: cfg.BlockInstanceMetdata,
+		Type:                     ECSENIPluginName,
+		CNIVersion:               client.cniVersion,
+		ENIID:                    cfg.ENIID,
+		IPV4Address:              cfg.ENIIPV4Address,
+		MACAddress:               cfg.ENIMACAddress,
+		IPV6Address:              cfg.ENIIPV6Address,
```

Comment:
Nit: Isn't the world a better place when struct field are initialized with the same order as they were defined? Now might be a good time to fix the ordering and move IPv6Address one line up :)

Reply:
ehh fair enough! put the IPV4 and IPV6 next to each other 😀